scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -21,54 +23,82 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
 	pop_after=max(0,pop_before-kill);
 	if (pop_after<=0) and (pop_before>0) then heres_after=0;
 
+	// We should also make the regular bombardment lower heresy, I'm gonna copy some bits of code from scr_purge_world 
+	// heres_before=max(star.p_heresy[planet]+star.p_heresy_secret[planet],star.p_influence[planet][eFACTION.Tau]);
+	// sci1=0;sci2=0;
+	// if (pop_before>0) then sci1=(pop_after/pop_before)*100;
+	// if (sci1>0) then sci2=min((sci1*2),action_score*2);
+	// heres_after=heres_before-sci2;
 
 
 	if (star_system.p_type[planet_number]!="Space Hulk"){
 	    var bombard_protection=1;
 	    switch(bombard_target_faction){
+			// case 1:
+				// txt2="##The Space Marine forces are difficult to bombard; ";
+				// bombard_protection=3;
+				// break;
 	    	case 2:
 	    		txt2="##The Imperial forces are suitably fortified; ";
-	    		break;
+				bombard_protection=2;
+	    		break; // I'm not sure about IG, maybe they should be left at 2, or, maybe they should be at 1, like the PDF
 	    	case 2.5:
 	    		if (star_system.p_owner[planet_number]<=5){
-	    			txt2="##The PDF forces are suitably fortified; ";
+	    			txt2="##The PDF forces are poorly fortified; ";
+					bombard_protection=1;
 	    		} else if (star_system.p_owner[planet_number]>5){
-	    			txt2="##The renegade forces are moderately fortified;" ;
+	    			txt2="##The renegade forces are poorly fortified; ";
+					bombard_protection=1;
 	    		}
-	    		break;
+	    		break; // I think PDF and renegades down there should be kind of poorly prepared for this
 	    	case 3:
-	    		txt2="##The Mechanicus forces are suitably fortified; ";
-	    		bombard_protection=0;
+	    		txt2="##The Mechanicus forces are well fortified; ";
+	    		bombard_protection=3; // If we get to Admech, I think they should be pretty capable with the hi-tech goodies they have
 	    		break;
+			// case 4:
+	    		// txt2="##The Inquisition forces are difficult to bombard; ";
+				// bombard_protection=3;
+	    		// break;
 	    	case 5:
 	    		txt2="##The Ecclesiarchy forces are concentrated within their Cathedral; ";
-	    		break;
+				bombard_protection=1;
+	    		break; // Maybe we should make it 0? Though, Cathedral does have a roof at least
 	    	case 6:
 	    		txt2="##The Eldar forces are challenging to pin down; ";
-	    		bombard_protection=3;
+	    		bombard_protection=4; // Hi-tech faction
 	    		break;
 	    	case 7:
-	    		txt2="##The Ork forces are well dug in; ";
-	    		bombard_protection=2;
+	    		txt2="##The Ork forces are poorly dug in; ";
+	    		bombard_protection=1; // I think they shouldn't be so resistant to it, though, Ideally we might want to make them more capable, if warboss is present.
```

Comment:
Yes, but I don't think it is that extreme, that it affects the game much.
I think ideally orks should be stronger if they have leaders in the location, like big mek (building contraptions, perhaps StOs too and raising resistance), weirdboy (Waaagh energy shenanigans) or warboss (higher effectiveness in general).
Otherwise, they should be mostly treated as feral orks, without effective leadership.

Reply:
@blogalus, just revert that protection on Orks and let's merge this already.

Reply:
Fine, if you say so.